build.gradle.kts

```diff
@@ -186,7 +186,7 @@ val checkGitAttributes by tasks.registering {
                 // Skip empty and comment lines.
                 .map { it.trim() }
                 .filter { it.isNotEmpty() && it.first() !in commentChars }
-                // The patterns is the part before the first whitespace.
+                // The patterns are located before the first whitespace.
```

Comment:
Actually, I believe this should get fixed in the opposite direction, and use singular "pattern" (because there can be only one), reading

> The pattern is the part before the first whitespace.